katas/content/single_qubit_measurements/distinguish_plus_and_minus/solution.md

```diff
@@ -1,8 +1,8 @@
-The input qubit is guaranteed to be either in basis state $|+\rangle$ or $|-\rangle$. This means that when measuring the qubit in the Pauli $X$ basis, the measurement will report the input state without any doubt. (Recall that these states are eigenstates for the Pauli $X$ matrix).  
+The input qubit is guaranteed to be either in basis state $|+\rangle$ or $|-\rangle$. This means that when measuring the qubit in the Pauli X basis, the measurement will report the input state without any doubt. (Recall that these states are eigenstates for the Pauli X matrix).
```

Comment:
Throughout the kata, it is just written as `Pauli X` without the math syntax. For consistency, I removed the math syntax here.